src/approaches/oracle_approach.py

```diff
@@ -55,3 +56,14 @@ def is_learning_based(self) -> bool:
 
     def _get_current_nsrts(self) -> Set[NSRT]:
         return self._nsrts
+
+    def get_option_model(self) -> _OptionModelBase:
+        """For ONLY an oracle approach, we allow the user to get the current
+        option model.
+
+        Note that this doesn't fit into the standard API for an
+        Approach, since solve() returns a policy, which abstracts away
+        the details of whether that policy is actually a plan under the
+        hood.
+        """
+        return self._option_model  # pragma: no cover
```

Comment:
remove this `pragma: no cover`, this should be easy to cover with a test in `test_oracle_approach.py`, just do like `assert isinstance(approach.get_option_model(), _OracleOptionModel)` somewhere